src/model/quotes.js

```diff
@@ -305,13 +338,15 @@ class QuotesModel {
           currencyId: quoteRequest.amount.currency
         })
 
-        // eslint-disable-next-line require-atomic-updates
-        refs.payerId = await this.db.createPayerQuoteParty(txn, refs.quoteId, quoteRequest.payer,
-          quoteRequest.amount.amount, quoteRequest.amount.currency)
-
-        // eslint-disable-next-line require-atomic-updates
-        refs.payeeId = await this.db.createPayeeQuoteParty(txn, refs.quoteId, quoteRequest.payee,
-          quoteRequest.amount.amount, quoteRequest.amount.currency)
+        ;[
+          refs.payerId,
+          refs.payeeId
+        ] = await Promise.all([
+          this.db.createPayerQuoteParty(txn, refs.quoteId, quoteRequest.payer,
+            quoteRequest.amount.amount, quoteRequest.amount.currency, payerEnumVals),
+          this.db.createPayeeQuoteParty(txn, refs.quoteId, quoteRequest.payee,
+            quoteRequest.amount.amount, quoteRequest.amount.currency, payeeEnumVals)
+        ])
```

Comment:
Putting off consolidating these two into a single insert since `createQuoteParty` has itself several queries that would also need refactoring to a single insert for the quote pair.